SPIFlashAnalyzer.py

```diff
@@ -104,7 +105,7 @@ def decode(self, frame: AnalyzerFrame):
                 diff = frame.start_time - self._last_time
             else:
                 diff = self._fastest_cs
-            diff = float(diff * 1_000_000_000)
+                diff = float(diff * 1_000_000_000)
```

Comment:
Why change this?

Reply:
I thought it was an indentation error because I'm used to having a newline after the body of an else statement. Sorry, good catch.